storage/src/main/java/org/apache/uniffle/storage/handler/impl/LocalFileWriter.java

```diff
@@ -33,10 +35,16 @@ public class LocalFileWriter implements FileWriter, Closeable {
   private FileOutputStream fileOutputStream;
   private long nextOffset;
 
+  @VisibleForTesting
   public LocalFileWriter(File file) throws IOException {
+    this(file, 8 * 1024);
+  }
+
+  public LocalFileWriter(File file, long bufferSize) throws IOException {
```

Comment:
Why you define long type but used as int type? I guess it is from the long type configure item, so you can convert to int type earlier, it is better to convert it after get value from config.

Reply:
BufferedOutputStream's constructor need int type and getSizeAsBytes will convert the string value to long type.
I will change the code to convert the value's type earlier.